### PR TITLE
Chore/update deps #trivial

### PR DESCRIPTION
**Pull request by kaisermann**
#### What does this PR do? \*

- Update tooling dependencies
- Update tooling configs
- Remove unneeded extra scripts

#### How to test it? \*

n/a

#### Describe alternatives you've considered, if any. \*

n/a

#### Related to / Depends on \*

n/a

### CODE REVIEW

File: react/package.json [hunk 1]
```diff
@@ -13,16 +13,15 @@
...
+    "@vtex/test-tools": "^3.1.0",
+    "@vtex/tsconfig": "^0.4.4",
     "apollo-cache-inmemory": "^1.6.5",
-    "eslint-config-vtex-react": "^6.1.1",
-    "eslint": "^6.8.0",
     "graphql": "^14.6.0",
     "prop-types": "^15.7.2",
-    "typescript": "3.7.3"
+    "typescript": "3.8.3",
+    "vtex.render-runtime": "http://vtex.vtexassets.com/_v/public/typings/v1/vtex.render-runtime@8.98.1/public/@types/vtex.render-runtime"
```
Comment (klzns) [inline react/package.json:25]: Should this be here?

File: react/package.json [hunk 2]
```diff
@@ -13,16 +13,15 @@
...
-    "@types/react": "^16.9.19",
-    "@vtex/test-tools": "^3.0.3",
-    "@vtex/tsconfig": "^0.4.3",
+    "@types/jest": "^25.1.4",
+    "@types/node": "^13.9.8",
+    "@types/react": "^16.9.31",
+    "@vtex/test-tools": "^3.1.0",
+    "@vtex/tsconfig": "^0.4.4",
     "apollo-cache-inmemory": "^1.6.5",
-    "eslint-config-vtex-react": "^6.1.1",
```
Comment (klzns) [inline react/package.json:22]: Will it find the package in the root directory?

File: react/package.json (hunk 1 above)
Comment (kaisermann) [reply react/package.json:25]: `vtex setup` thingie 🤷‍♂

Comment (klzns) [reply react/package.json]: I would remove it, since it doesnt work properly

File: react/package.json (hunk 2 above)
Comment (kaisermann) [reply react/package.json:22]: Yep. We're concentrating the tooling packages in the root

**Pull request merged**